tests/failing.llvm

```diff
@@ -126,8 +73,6 @@ BlockchainTests/GeneralStateTests/Pyspecs/cancun/eip6780_selfdestruct/self_destr
 BlockchainTests/GeneralStateTests/Pyspecs/cancun/eip6780_selfdestruct/selfdestruct_pre_existing.json,*
 BlockchainTests/GeneralStateTests/Pyspecs/cancun/eip7516_blobgasfee/blobbasefee_before_fork.json,*
```

Comment:
these two tests still fail due to lack of support for transition tests in the KEVM test harness.